vars/buildPlugin.groovy

```diff
@@ -172,6 +173,9 @@ def call(Map params = [:]) {
                                 }
                                 checkstyle(cp)
                             }
+                            if (isMaven && archiveJacoco) {
```

Comment:
does warnings-ng cover this or do both need to be used?

Reply:
shouldn't this be the code coverage api plugin?
https://plugins.jenkins.io/code-coverage-api/

Reply:
See #153